documentation/modules/ROOT/examples/orgs_vars/superadmin/env/dev/controller_credentials.d/controller_credentials_scm.yml

```diff
@@ -5,9 +5,7 @@ controller_credentials:
     credential_type: "Source Control"
     organization: "{{ orgs }}"
     inputs:
-      username: "<CHANGE_ME>"
+      username: "${GITLAB_USER}"
```

Comment:
I'd prefer to use `<CHANGE_ME>` instead of leaving sensitive information into environment variables.

Reply:
same